web/docs/deployment/deployment-methods/cli.md

```diff
@@ -311,16 +397,159 @@ You can find the list of all available Fly regions by running:
 flyctl platform regions
 ```
 
-### Multiple Fly.io Organizations
+#### Multiple Fly.io Organizations
 
 If you have multiple organizations, you can specify a `--org` option. For example:
 
 ```shell
 wasp deploy fly launch my-wasp-app mia --org hive
 ```
 
-### Building Locally
+#### Building Locally
 
 Fly.io offers support for both **locally** built Docker containers and **remotely** built ones. However, for simplicity and reproducibility, the CLI defaults to the use of a remote Fly.io builder.
 
 If you want to build locally, supply the `--build-locally` option to `wasp deploy fly launch` or `wasp deploy fly deploy`.
+
+### Railway CLI Commands {#railway-cli}
+
+#### `launch` command
+
+`launch` is a convenience command that runs `setup` and `deploy` in sequence.
+
+```shell
+wasp deploy railway launch <project-name>
+```
+
+It accepts the following arguments:
+
+- `<project-name>` - the name of your project <Required />
+
+It gives you the same result as running the following commands:
+
+```shell
+wasp deploy railway setup <project-name>
+wasp deploy railway deploy <project-name>
+```
+
+##### Explicitly providing the Railway project ID
+
+Wasp CLI tries to create a new Railway project named `<project-name>`, but you can use an existing project you have created on Railway by passing the `--existing-project-id [projectId]` option.
+
+##### Environment Variables {#railway-launch-environment-variables}
+
+###### Server
```

Comment:
IMO Such deep nesting is reflecting the fact that we might need to split this page.